unitTests/ui/ResponsiveComponents/ResponsiveRecommendationTest.ts

```diff
@@ -70,6 +71,26 @@ export function ResponsiveRecommendationTest() {
       expect(responsiveDropdownContent.positionDropdown).toHaveBeenCalled();
     });
 
+    describe('when it should switch to small mode but the responsiveMode is on medium (or large)', () => {
+      it('returns false when needDropdownWrapper is called', () => {
+        new SearchInterface(root.el, { responsiveMode: 'medium' });
+        responsiveRecommendation = new ResponsiveRecommendation(root, '', { dropdownHeaderLabel: 'label' }, responsiveDropdown);
+        shouldSwitchToSmallMode();
```

Comment:
The `should` in the function name suggests to me that I will get back a boolean, but it seems to be a `void` function. Instead, would `switchToSmallMode()` be a better name?